aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -656,6 +708,25 @@ module aptos_framework::fungible_asset {
         move_to(&metadata_object_signer, supply);
     }
 
+    public fun upgrade_store_to_concurrent(
+        ref: &ExtendRef,
```

Comment:
Unfortunately the requirement of an `ExtendRef` blocks upgrades in the general case, most importantly for wallet transfers.

I've solved this in https://github.com/aptos-labs/aptos-core/pull/12348